data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -560,6 +574,24 @@ def build_filing_json_meta_data(filing_type: str, filing_subtype: str, effective
         'colinDisplayName': get_colin_display_name(data)
     }
 
+    if raw_filing_type == 'conversion':
+        # will populate state filing info for conversion in the following steps
+        # based on converted filing type and converted filing subtype
+        if filing_type in LEAR_STATE_FILINGS:
+            state_change = True
+        else:
+            state_change = False
+        filing_json['filing']['conversion'] = {
+            'convFilingType': filing_type,
+            'convFilingSubType': filing_subtype,
+            'stateChange': state_change
+        }
+        meta_data['conversion'] = {
+            'convFilingType': filing_type,
+            'convFilingSubType': filing_subtype,
+            'stateChange': state_change
+        }
```

Comment:
So this is how we representing changes in `meta_data` for modernized system.  Maybe do something similar?  i.e. use `fromState` and `toState`

Here is an example for alteration in Test for BC1062617.
```
{
  "alteration": {
    "toLegalType": "BC",
    "fromLegalType": "BEN"
  },
  "legalFilings": [
    "alteration"
  ],
  "applicationDate": "2025-01-30T23:32:46.529519+00:00"
}
```

Another alteration example in Test for BC1061691.

```
{
  "alteration": {
    "toLegalName": "ALTERS ULC",
    "toLegalType": "ULC",
    "fromLegalName": "1061691 B.C. LTD.",
    "fromLegalType": "BC"
  },
  "legalFilings": [
    "alteration"
  ],
  "applicationDate": "2025-01-31T20:26:02.809366+00:00"
}
```

Reply:
This is what conversion `meta_data` looks like...
```json
{
  "conversion": {
    "stateChange": true,
    "convFilingType": "dissolution",
    "convFilingSubType": "unknown"
  },
  "dissolution": {
    "dissolutionDate": "2000-01-01",
    "dissolutionType": "unknown"
  },
  "colinFilingInfo": {
    "eventId": 123,
    "eventType": "CONVDSO",
    "filingType": null
  },
  "colinDisplayName": null,
  "isLedgerPlaceholder": true
}
```

the `conversion` field will be always populated but the converted filing name field will not always be populated, and it exists only when the converted filing is `restoration` or `dissolution` now.